Controller/ElasticsearchProxyController.php

```
@@ -31,7 +31,7 @@ public function proxyAction(Request $request, $index, $slug)
         $dispatcher->dispatch('elasticsearch_proxy.before_elasticsearch_request', $event);
         $data = $event->getQuery();
         // Get url for elastic search
-        $url = $this->getElasticSearchUrl($request->getQueryString(), $index, $slug);
+        $url = $this->getElasticSearchUrl($request->getQueryString(), $index, $slug, $container);
```

PHPStan (2.2.8)

[warning] Parameter `#1` $queryStr of method Xola\ElasticsearchProxyBundle\Controller\ElasticsearchProxyController::getElasticSearchUrl() expects string, string|null given.


(argument.type)